packages/frontend/src/pages/Periods/Details/components/ReceiverTable.tsx

```diff
@@ -39,10 +40,22 @@ const ReceiverTable = (): JSX.Element | null => {
       ],
       []
     );
+    const data = period?.receivers
+      ? sortBy(period.receivers, [
```

Comment:
`react-table` has built in sorting. See a few lines below. 

But, perhaps we should try to move away from `react-table` where it is not actually motivated?